tests/unit/bokeh/models/widgets/test_DatePicker.py

```diff
@@ -0,0 +1,61 @@
+#   -----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2020, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+import pytest ; pytest
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Module under test
+from bokeh.models.widgets import DatePicker
+
+#-----------------------------------------------------------------------------
+# Setup
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+
+class TestDatePicker:
+    def test_setting_value_as_date_time(self) -> None:
```

Comment:
I'm not sure "as_date_time" is an accurate description here, suggest "as_iso_date_format" instead